.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: ['2.7', '3.0', '3.1', '3.2', '3.3']
```

Comment:
I wouldn't worry too much about a matrix, tbh! Feel free to just stick to Ruby 3.3, I don't think I've ever seen compatibility issues with Ruby versions for this gem